README.md

```diff
@@ -10,26 +10,201 @@ Routing solution to link URLs to Metal.js components using HTML5 History API.
 
 ## Setup
 
-1. Install NodeJS >= [v0.12.0](http://nodejs.org/dist/v0.12.0/), if you don't have it yet.
+1. Install a recent release of [NodeJS](https://nodejs.org/en/download/) if you
+don't have it yet.
 
-2. Install global dependencies:
+2. Install local dependencies:
 
   ```
-  [sudo] npm install -g gulp
+  npm install
   ```
 
-3. Install local dependencies:
+3. Run the tests:
 
   ```
-  npm install
+  npm test
   ```
 
 4. Build the code:
 
   ```
-  gulp build
+  npm run build
   ```
 
 ## Demo
 
-  Check it out the demo page by running `npm run demo`.
+  Check out the demo page by running `npm run demo`.
+
+## Use
```

Comment:
`Use` should be the first section, following what we did in [metal-ajax](https://github.com/metal/metal-ajax#use) and the rest last week.